### DISCUSSION TITLE
Unraid & Home Assistant - Best Setup

### DISCUSSION THREAD

**Discussion by sebastian-tischer** [blakeblackshear/frigate#8157] [General]
Hi Blake,

I need your help (again 😥 )...
In preparation of my new Setup (HPE Microserver Gen10+ v2 with Xeon processor + Nvidia T400 GPU + Google Coral), I think about the best way to install Frigate.

Option 1:
- Home Assistant as VM
- Frigate as Addon in HASS
- Processing of images and everything happens with Coral + GPU in HASS VM
=> Is it possible for HASS VM and Frigate as Addon to leverage the GPU + Coral?

Option 2:
- Home Assistant as VM
- Frigate as Docker in Unraid
- Transcoding/Encoding/Decoding by GPU + Coral happens in Docker (maybe Nvidia driver needed for installation)
- HASS only uses the HACS integration to get images / streams
=> With that way, can I also see the events in HASS?

What is your suggested way of integration?

Thanks and warm regards
Sebastian

**Comment by NickM-27** [comment]
Nvidia GPU is not supported in addons (as shown in the docs) and generally running in native docker is recommended

**Reply by sebastian-tischer** [reply]
Yeah I read that and stumbled upon this 😞 

When running Frigate as Docker (as recommended), can I see events of detections in HASS?

**Reply by NickM-27** [reply]
Not sure what you mean by see events, that is quite ambiguous. But the clips and snapshots are in the HA media page and you can also receive notifications via MQTT

No features are lost when having frigate externally.

**Reply by sebastian-tischer** [reply]
I love the tight integration right now, because I can quickly adjust config, jump to cam and check the event tab in Frigate in HASS.
So I can replay media files, see which person was recognized etc.

This integration in the sidebar or seeing the "events" is lost right? Or if not, where can I see that in HASS?

<img width="1674" alt="Bildschirmfoto 2023-10-13 um 23 32 58" src="https://github.com/blakeblackshear/frigate/assets/63227608/930f1cd0-516c-4f25-9081-27722077c6ba">

**Reply by NickM-27** [reply]
You just install the frigate proxy addon

**Reply by sebastian-tischer** [reply]
Fuck dude, you rock... 👍  Thats amazing - so frigate will run as docker and all will be perfect - without stupid questions from me...
